magento/magento2
# Report Handled Exceptions To New Relic

### Description

Currently, if you're using New Relic you'll know about fatal errors via the APM instrumentation, but handled exceptions will not be reported. This change reports handled exceptions to New Relic

### Manual testing scenarios

1. Configure the New Relic PHP agent in your environment and enable the New Relic module
2. Purposely enter the wrong database credentials in `app/etc/env.php`.
3. Notice the exception being reported to New Relic

### Contribution checklist
